server/README.md

```diff
@@ -3,88 +3,28 @@ Introduction
 
 This describes the server API. For installation instructions, see the README file at the root directory level.
 
-User API
+API documentation
 ===
 
-These APIs allow front-end to get current user or login a user.
+Once you have configured the api server and nginx, you can browse api endpoints at http://optima.dev/api/spec.html (click show/hide)
 
-* `/api/user/current`
+These APIs allow front-end to get current user or login a user
 
-  Returns `401 Unauthorized` if user is not logged in. Otherwise this JSON:
-
-  ```
-  {
-	email: "iwein@startersquad.com"
-	name: "Iwein Fuld"
-   }```
-
-* `/api/user/create`
-
-  Following data is posted when creating a new user:
-
-  ```
-  {
-	email: "iwein@startersquad.com"
-	name: "Iwein Fuld"
-	password: 'whatever'
-   }```
-
-  On success, user is sent back this JSON:
-
-  ```
-  {
-     "email": "iwein@startersquad.com",
-     "name": "Iwein Fuld"
-   }```
-
-   On name already used, user is sent back this JSON:
-
-   ```
-   {
- 	 status: "Username in use"
-    }```
-
-* `/api/user/login`
-
-  Following data is posted when doing login:
-
-  ```
-	email: "iwein@startersquad.com"
-	password: "whatever"
-  ```
-
-  On successful login, user is redirected to the home page.
-
-  On login error, a 401 Unauthorized response is returned.
+Celery
+===
 
-* `/api/user/logout`
+In order to run celery on your local machine, you need to have redis installed.
 
-  User is logged out and redirected back to the login page.
+On Ubuntu:
+---
 
-Project API
-=====
+`sudo apt-get install redis-server`
 
-These APIs allow front-end to work with projects.
+On MacOS X:
+---
 
-* `/api/project/info`
+Follow [these instructions](http://jasdeep.ca/2012/05/installing-redis-on-mac-os-x/)
```

Comment:
please also update the main README.md for anything essential -- the subreadmes should be additional info, but the README in the root dir should be sufficient to get up and running